dataset/dictionary/characters.json5

```diff
@@ -2379,22 +2380,26 @@
   {
     en: "Parvezravan",
     ja: "パーヴェズラヴァン",
+    zhCN: "帕维兹拉万",
     tags: [ "sumeru", "character-sub" ],
     notes: "世界任務「ビルキースの哀歌」で言及されるキャラクター。「パーヴェズラヴァン陛下」は \"His Majesty Parvezravan\"",
     examples: [{
       en: "...It is hard to believe that His Majesty Parvezravan may be bewitched by spectacles conjured by Jinn slaves, and yet he cannot see the foundations of the very earth coming loose under his feet...",
       ja: "…信じられない。パーヴェズラヴァン陛下は知恵のないジンニーの奴隷たちが作った奇観に夢中になり、足元で緩くなっていく地盤に少しも気づかないとは…",
+      zhCN: "…难以置信，帕维兹拉万陛下沉迷于无智的镇灵奴隶们所搭建的奇观，但却对脚下已然松动的地基毫无察觉… ",
```

Comment:
Unfortunately, Chinese example sentences are not displayed on the website.
I have created an issue to show the Chinese examples (xicri/genshin-dictionary#84), but it may need time to fix this.
However, I will keep this Chinese example for future updates.